app/models/user.rb

```diff
@@ -539,14 +539,13 @@ def unread_notifications_of_priority(high_priority:)
   end
 
   MAX_UNREAD_HIGH_PRI_BACKLOG = 200
```

Comment:
Can we rename this constant now that it's no longer for high priority notifications only? Also I'd bump the constant up to say 400 for the same reason.